library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/utils/ShapeWireframeForgeryFactory.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.utils
+
+import com.datadog.android.sessionreplay.model.MobileSegment
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.ForgeryFactory
+
+internal class ShapeWireframeForgeryFactory :
+    ForgeryFactory<MobileSegment.Wireframe.ShapeWireframe> {
+    override fun getForgery(forge: Forge): MobileSegment.Wireframe.ShapeWireframe {
+        return MobileSegment.Wireframe.ShapeWireframe(
+            id = forge.aLong(min = 0),
+            x = forge.aLong(min = 0),
+            y = forge.aLong(min = 0),
+            width = forge.aLong(min = 1),
+            height = forge.aLong(min = 1),
+            shapeStyle = forge.aNullable {
+                MobileSegment.ShapeStyle(
+                    backgroundColor = forge.aNullable {
+                        forge.aStringMatching("#[0-9A-F]{6}FF")
+                    },
+                    opacity = forge.aFloat(min = 0f, max = 1f),
+                    cornerRadius = forge.aLong()
```

Comment:
why here we use `forge.aLong()`, but in `TextWireframeForgeryFactory` it is `forge.anInt(min = 0)` for the similar argument?

Reply:
you're right, I will fix those.